js/view/sideBarView.js

```diff
@@ -36,10 +36,20 @@ var SideBarView = function (container, model) {
 
   this.confirmationButton = container.find("#confirmationButton");
 
+  this.update = function(model, changeDetails) {
+    if (changeDetails== "numberOfGuests") {
+      var num = model.getNumberOfGuests();
+      numberOfGuests.html(num);
+    }
+  }
+
+  model.addObserver(this.update);
```

Comment:
Maybe we should add Observers in Controller.js

Reply:
But it's okay for now I think.

Reply:
Here they seem to do it in the View: https://kth.instructure.com/courses/7528/pages/observer?module_item_id=114607

Reply:
> Here they seem to do it in the View: https://kth.instructure.com/courses/7528/pages/observer?module_item_id=114607

Good, I will find some time to read through it. But I think that is because their "controllers' are actually their views. Whilst, ours are separated.